e2e/tests/e2e_happy_path/HappyPath.spec.ts

```diff
@@ -147,8 +147,13 @@ suite('Language server validation', async () => {
 suite('Validation of workspace build and run', async () => {
     test('Build application', async () => {
         await runTask('che: build-file-output');
-        await projectTree.expandPathAndOpenFileInAssociatedWorkspace(projectName, 'build-output.txt');
-        await editor.followAndWaitForText('build-output.txt', '[INFO] BUILD SUCCESS', 300000, 10000);
+
+        //workaround for issue: https://github.com/eclipse/che/issues/14771
+        
+        // await projectTree.expandPathAndOpenFileInAssociatedWorkspace(projectName, 'build-output.txt');
+        await projectTree.expandPathAndOpenFileInAssociatedWorkspace(projectName, 'result-build-output.txt', 220000);
+        await editor.waitText('result-build-output.txt', '[INFO] BUILD SUCCESS');
+        // await editor.followAndWaitForText('build-output.txt', '[INFO] BUILD SUCCESS', 300000, 10000);
```

Comment:
@musienko-maxim Why did you left commented code? This is not a very good approach.